tests/analyze.R

```diff
@@ -106,20 +105,45 @@ ProcessAll = function(ctx) {
                   proportion = rappor$proportion,
                   dist = "rappor")
 
-  # Fill in zeros for values missing in RAPPOR.  It makes the ggplot bar plot
-  # look better.
+  # Fill in zeros for values missing in RAPPOR and in actual data
```

Comment:
rappor isn't always a subset of actual. Remember the false positive for v57 or something that used to show up for one of the distributions? We have to be more structured about detecting and reporting false positives, but this is a first step.